other_custom_types/src/lib.rs

```diff
@@ -199,4 +199,18 @@ impl CustomTypesContract {
     pub fn tuple_strukt(_env: Env, tuple_strukt: TupleStruct) -> TupleStruct {
         tuple_strukt
     }
+
+    pub fn get_object_vec_option(env: Env) -> Option<Vec<Test>> {
```

Comment:
For consistency with other functions in this contract, we should drop the `get_` prefix and use the same names as other functions here.
```suggestion
    pub fn option_vec_strukt(env: Env) -> Option<Vec<Test>> {
```